app/src/main/res/xml/debug_settings.xml

```diff
@@ -49,6 +49,11 @@
         android:title="@string/show_image_indicators_title"
         app:iconSpaceReserved="false" />
 
+    <Preference
+        android:key="@string/check_new_streams_key"
+        android:title="@string/check_new_streams"
+        app:iconSpaceReserved="false" />
+
```

Comment:
Why is this only available for debugging purposes?
I think it would also fit very good onto the notification-settings.

Reply:
One can trigger a notification check manually by refreshing the feeds. I don't think this needs to be in the notification settings. This setting is useful for debugging but does not belong in the general notication settings IMO.

Reply:
> One can trigger a notification check manually by refreshing the feeds.

Oh I didn't know that. Maybe we should add a not to that somewhere.

Reply:
Refreshing a feed does not show the notification, but just refreshes the feed. But the new items will be shown in the list (and will be highlighted with #7050).
Not sure, where you want to put a note about this.